content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,382 @@
+
+As each block is finalized, AFD will attempt to promote _expired accusations_ (where the _innocence proof submission window_ has elapsed) to proven _faults_.
+<!--
+To check if an _offending_ validator has a _pending accusation_, a _reporting_ validator calls protocol functions:
+
+- [`canAccuse()`](/reference/api/accountability/#canaccuse): to determine if (a) an offending validator is accusable, and, (b) the _deadline_ number of blocks remaining in the _innocence proof submission window_ if there is a _pending accusation_.
+- [`canSlash()`](/reference/api/accountability/#canslash): to determine if the  _offending_ validator already has a _fault_ for a rule infraction in the epoch with a _severity_ higher than the new accusable rule infraction detected.
+-->
+
+#### Innocence
+
+An _innocence_ is a claim refuting an _accusation_ for an accusable rule infraction. An _innocence_ is made by an _offending_ validator to prove innocence from a _pending accusation_ against it. Claims of _innocence_ are made by submitting _proof_ of innocence of an accusable rule infraction on-chain as an event of type `Innocence`. If successful, a proof of innocence cancels the _pending accusation_.
+
+The _offending_ validator has a limited time window to submit a _proof_ of _innocence_, otherwise, the _accusation_ may be _promoted_ to a _fault_ and be slashable end of epoch.
+
+An _innocence_ must be submitted within an _innocence proof submission window_ to be accepted, a designated number of blocks set as a protocol parameter. The window begins at the block height number at which the _accusation_ was reported.
+
+If the _innocence_ is successfully verified, then the _accusation_ queue is checked and the corresponding _accusation_ is cancelled. The _pending accusation_ state is cleared and a _reporting_ validator is now able to submit a _new accusation_ against the _offending_ validator.
+<!--
+To check if a _new accusation_ has been made against it, an _offending_ validator:
+
+- subscribes to `NewAccusation` events where it is the offender, retrieves the accountability event ID and queries the `Events` data structure to retrieve the _pending accusation_ against it.
+-->
+After successful [handling and verification](/reference/api/aut/op-prot/#handleevent-accountability-contract) of an _innocence_ on-chain an `Innocence` proof event is emitted logging: _offending_ [validator identifier](/concepts/validator/#validator-identifier) address, and `0` indicating there are no pending accusations against the validator.
+
+{{% alert title="Note" %}}
+As noted under [Accusations](/concepts/accountability/#accusations) above, a validator with a _pending accusation_ is not accusable because protocol has to wait to determine if the _pending accusation_ has been defended or, if not, promoted to a fault or not. Until then, protocol cannot determine if the _offending_ validator has committed a rule infraction with a higher _severity_ or not in the epoch.
```

Comment:
```suggestion
As noted under [Accusations](/concepts/accountability/#accusations) above, a validator with a _pending accusation_ is not accusable because protocol has to wait to determine if the _pending accusation_ has been defended or, if not, promoted to a fault. Until then, protocol cannot determine if the _offending_ validator has committed a rule infraction with a higher _severity_ or not in the epoch.
```